resource/login.php

```diff
@@ -0,0 +1,70 @@
+<?php
+    require 'vendor/autoload.php';
+
+    use Lcobucci\JWT\Builder;
+    use Lcobucci\JWT\JwtFacade;
+    use Lcobucci\JWT\Signer\Hmac\Sha256;
+
+    // Establish connection with database
+    require('database.php');
+    require('utils.php');
+    require('secret.php');
+
+    // Parse User object
+    $json = file_get_contents('php://input');
+    $data = json_decode($json);
+
+    $username = mysqli_real_escape_string($dbconn, stripslashes($data->username));
+    $password = mysqli_real_escape_string($dbconn, stripslashes($data->password));
+
+    // Attempt to locate User in database
+    if(loginAlreadyExists($dbconn, $username))
+    {
+        // Get password for this user from database
+        $stmt = $dbconn->prepare("SELECT Password, userID FROM Users WHERE username=?");
+        $stmt->bind_param("s", $username);
+        $stmt->bind_result($out_password, $out_userID);
+        $stmt->execute();
+        $stmt->fetch();     // Get first result from statement's execution
+        $stmt->close();
+
+        // echo nl2br("Stored password for user ".$username." (ID: ".$out_userID."): ".$out_password."\n");
+
+        // Compare passwords
+        if(password_verify($password, $out_password))
```

Comment:
Is there going to be a later PR where it adds the token to the database so we can actually pull it out and determine the user based on the token? Or is there some part of JWT that we can use to do that without the database?

Reply:
The user ID is embedded in the token, so it is not necessary to add it to the database.

Reply:
Perfect!